paddle/framework/tensor.cc

```diff
@@ -0,0 +1,32 @@
+#include "paddle/framework/tensor.h"
+
+namespace paddle {
+namespace framework {
+
+int Tensor::Rank() const { return arity(dims_); }
+
+int Tensor::Numel() const { return product(dims_); }
+
+void Tensor::Resize(const DDim& dims) {
```

Comment:
Do you mean that if users want to change tensor's size, they can invoke `mutable(DDim, Place)` directly?